airflow/www/app.py

```diff
@@ -148,7 +149,9 @@ def create_app(config=None, testing=False):
         init_plugins(flask_app)
         init_connection_form()
         init_error_handlers(flask_app)
-        init_api_connexion(flask_app)
+        if conf.get("webserver", "run_internal_api"):
+            init_api_connexion(flask_app)
+        init_api_internal(flask_app)
```

Comment:
I am not sure what you tried to achieve here but it seems you are disabling the Rest API if `run_internal_api` is `False`
```suggestion
        if conf.get("webserver", "run_internal_api"):
            init_api_internal(flask_app)
        init_api_connexion(flask_app)
```